static/bidder-params/openx.json

```diff
@@ -22,9 +23,10 @@
       "format": "uuid"
     },
     "customFloor": {
-      "type": "number",
+      "type": ["number", "string"],
       "description": "The minimum CPM price in USD.",
-      "minimum": 0
```

Comment:
should add json tests 
- to have request where unit is string 
- to have request where unit is number 
- to have request where customFloor is string 
- - to have request where customFloor is number

Reply:
These parameter variations are in `optional-params.json`:
`unit` is string: https://github.com/prebid/prebid-server/pull/3668/files#diff-4214c2dc95f1c24891a70e424d9576ab40f10447745b95f68a877ca7145f1bd5R12
`unit` is number: https://github.com/prebid/prebid-server/pull/3668/files#diff-4214c2dc95f1c24891a70e424d9576ab40f10447745b95f68a877ca7145f1bd5R28
`customFloor` is string: https://github.com/prebid/prebid-server/pull/3668/files#diff-4214c2dc95f1c24891a70e424d9576ab40f10447745b95f68a877ca7145f1bd5R15
`customFloor` is number: https://github.com/prebid/prebid-server/pull/3668/files#diff-4214c2dc95f1c24891a70e424d9576ab40f10447745b95f68a877ca7145f1bd5R31